### ISSUE TITLE
certain notebooks don't work at all

### ISSUE THREAD

**Issue by schlichtanders** [computationalmodelling/nbval#55]
when executing nbval on one of my notebooks, I get weird py.test --nbval reportings like the following:

```
<<<<<<<<<<<< Reference output from ipynb file:
stream
============ disagrees with newly computed (test) output:  
streamstreamstreamstreamstreamstreamstreamstreamstreamstreamstreamstreamstreamstreamstream
>>>>>>>>>>>>
```
```
<<<<<<<<<<<< Reference output from ipynb file:
stream
============ disagrees with newly computed (test) output:  
streamstreamstream
>>>>>>>>>>>>
```
```
<<<<<<<<<<<< Reference output from ipynb file:
stream
============ disagrees with newly computed (test) output:  
streamst...<snip base64, md5=1d90ad9225fdb1ef...>
>>>>>>>>>>>>
```

I was not able to make a minimal working example out of it yet, however maybe someone has seen this before and can help me.

[Interestingly, when downloading the example documentation.ipynb from nbval repository, everything works perfect]

**Comment by vidartf** [comment]
Seems to be a duplicate of #54?

**Comment by vidartf** [comment]
Fix suggested in #57.

**Comment by takluyver** [comment]
Sorry, I thought #57 was the one we already merged.

**Comment by vidartf** [comment]
Fixed in #57.

**Comment by schlichtanders** [comment]
thank you so much! awesome
